AudreyW_Lab_1.ipynb

```diff
@@ -0,0 +1,598 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ax = sns.distplot(data_poisson1,\n",
+    "                  bins=30,\n",
+    "                  kde=True,\n",
+    "                  color='skyblue',\n",
+    "                  hist_kws={\"linewidth\": 15,'alpha':1})\n",
+    "ax.set(xlabel='Poisson Distribution', ylabel='Frequency')"
```

Comment:
Your graphs all look super clean, however I believe we are supposed to plot the y axis on a log scale rather than a linear scale so be sure to double check they all have that feature